src/controllers/ArticleLikeController.js

```diff
@@ -0,0 +1,63 @@
+import status from '../config/status';
+import db from '../models';
+import { dbFindSingle, dbUpdate, dbFindAll } from '../helpers/queryHelper';
+
+/**
+ * A class to control the Article likes
+ */
+export default class ArticleLikeController {
+  /**
+   * A method to create an article like
+   * @param {object} req the request to get like
+   * @param {object} res the response of like from server
+   * @returns {object} the return after liking a particular article
+   */
+  static async like(req, res) {
+    const userId = req.user.id;
+    const { articleSlug } = req.params;
+    const condition = { userId, articleSlug };
+    const findLike = await dbFindSingle(db.ArticleLike, condition);
+    if (findLike) {
```

Comment:
we could have a reusable if check that passes the message you want to be displayed.